aries_vcx/src/messages/connection/service.rs

```diff
@@ -20,7 +49,7 @@ impl Service {
     }
 
     pub fn set_routing_keys(mut self, routing_keys: Vec<String>) -> Self {
-        self.routing_keys = Some(routing_keys);
+        self.routing_keys = routing_keys;
```

Comment:
generally speaking routing keys can be undefined, but if this makes things easier for now, I don't mind the change

Reply:
It is not strictly necessary, but it does simplify the code slightly and we don't have a use-case for undefined routing keys ATM AFAIK